test/functional/app/api/test_user_api.py

```diff
@@ -4,11 +4,65 @@
 
 from server.utils.auth import get_complete_auth_token
 from server.utils.phone import proccess_phone_number
+from server.models.transfer_usage import TransferUsage
 
 fake = Faker()
 fake.add_provider(phone_number)
 
 
+@pytest.mark.parametrize("phone, business_usage_name, referred_by, tier, status_code", [
+    (None, 'Fuel/Energy', '+61401391419', 'superadmin', 400),
+    (fake.msisdn(), 'Fuel/Energy', fake.msisdn(), 'superadmin', 200),
+    (fake.msisdn(), 'Food/Water', fake.msisdn(), 'view', 403)
+])
+def test_create_user(test_client, authed_sempo_admin_user, init_database, phone,
+                     business_usage_name, referred_by, tier, status_code):
+    if tier:
+        authed_sempo_admin_user.set_held_role('ADMIN', tier)
+        auth = get_complete_auth_token(authed_sempo_admin_user)
+    else:
+        auth = None
+
+    response = test_client.post(
+        "/api/v1/user/",
+        headers=dict(
+            Authorization=auth,
+            Accept='application/json'
+        ),
+        json={
+            'first_name': 'John',
+            'last_name': 'Smith',
+            'bio': 'EasyMart',
+            'gender': 'female',
+            'phone': phone,
+            'is_vendor': False,
+            'is_tokenagent': False,
+            'is_groupaccount': False,
+            'initial_disbursement': 400,
+            'location': 'Elwood',
+            'business_usage_name': business_usage_name,
+            'referred_by': referred_by
+        })
+
+    assert response.status_code == status_code
+    if response.status_code == 200:
+        data = response.json['data']
+        assert isinstance(data['user'], object)
+        assert data['user']['first_name'] == 'John'
+        assert data['user']['last_name'] == 'Smith'
+        assert data['user']['custom_attributes']['bio'] == 'EasyMart'
+        assert data['user']['custom_attributes']['gender'] == 'female'
+        assert data['user']['phone'] == proccess_phone_number(phone)
+        assert data['user']['is_vendor'] is False
+        assert data['user']['is_tokenagent'] is False
+        assert data['user']['is_groupaccount'] is False
+        assert data['user']['transfer_accounts'][0]['balance'] == 400
```

Comment:
This assertion is breaking in CircleCI

Reply:
yeah it's werid because it passes locally ..

Reply:
should be fixed :)